examples/tutorial/visuals/05_viewer_location.py

```diff
@@ -0,0 +1,98 @@
+# vispy: testskip
+"""
+   Tutorial: Creating Visuals
+      05. Camera location
+--------------------------------
+
+In this tutorial we will demonstrate how to determine the direction from which
+a Visual is being viewed. 
```

Comment:
I get a warning in this example that would be better to avoid:
```
WARNING: Variable a_normal is not an active attribute
```
Is there some way we could modify the mesh visual so that providing the normals is optional (without causing such warnings)?